web/src/main/java/co/mobileaction/example/web/repository/IPostRepository.java

```diff
@@ -13,4 +13,6 @@
 public interface IPostRepository extends JpaRepository<Post, Long>
 {
     List<Post> findAllByUserId(Long userId);
+
+    void deleteByUserId(Long userId);

```

Comment:
please check hibernate logs. There is more efficent way to implement this method. Consider to use @Query